config/application.rb

```diff
@@ -50,5 +50,7 @@ class Application < Rails::Application
     end
 
     config.active_storage.service = (Settings&.active_storage&.service.presence || "local").to_sym
+
+    config.action_view.field_error_proc = Proc.new { |html_tag, instance| "<span class='field_with_errors'>#{html_tag}</span>".html_safe }
```

Comment:
This overrides the default `field_error_proc`. The reason the white space was being inserted was because the `field_error_proc` defaults to wrapping the error elements in  separate `<div>`s which then caused the label and input field to be rendered as separate html blocks. This changes the behavior to generate the `field_with_errors` element as a `<span>` so that it renders inline.

Another potential solution would be, if we aren't utilizing the `field_with_errors` class, to disable the `field_error_proc` completely, so that it just returns the original html. If we don't want to override `field_error_proc`, we could maybe add `.field_with_errors { display: inline; }` to the main css file.